### PR TITLE
make it better (by removing everything about Dmoj) because mcpt ftw p.s. dont let them be mean

### PR DESCRIPTION
**Pull request by aamri2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-DMOJ: Modern Online Judge [![Build Status](https://github.com/DMOJ/online-judge/workflows/build/badge.svg)](https://github.com/DMOJ/online-judge/actions/) [![Slack](https://slack.dmoj.ca/badge.svg)](https://slack.dmoj.ca)
+WLMOJ: Modern Online Judge [![Build Status](https://github.com/DMOJ/online-judge/workflows/build/badge.svg)](https://github.com/mcpt/wlmoj/actions/)
```
Comment (Ninjaclasher) [inline README.md:1]: Badge still points to the DMOJ link. Also, remove the `: Modern Online Judge`

File: README.md [hunk 2]
```diff
@@ -30,10 +30,6 @@ See it live at [dmoj.ca](https://dmoj.ca/)!
 * OAuth login with Google, Facebook, and Github
 * Two-factor authentication support
 
-## Installation
```
Comment (Ninjaclasher) [inline README.md:33]: Keep the installation process, even if it links to the DMOJ docs.

File: README.md [hunk 3]
```diff
@@ -89,17 +85,9 @@ Use the site in whatever language you're most comfortable in &mdash; currently f
...
 
 
-### Highly featured administration interface
-The DMOJ admin interface is highly versatile, and can be efficiently used for anything from managing users to authoring problem statements.
-
-
-![](https://dmoj.ml/data/_other/readme/problem-admin.png)
-
-![](https://dmoj.ml/data/_other/readme/admin-dashboard.png)
-
```
Comment (Ninjaclasher) [inline README.md:99]: Why are these deleted?

File: README.md [hunk 4]
```diff
@@ -89,17 +85,9 @@ Use the site in whatever language you're most comfortable in &mdash; currently f
...
 ## Supported Languages
 
-Check out [**DMOJ/judge**](https://github.com/DMOJ/judge) for more judging backend details.
+Check out [**mcpt/WLMOJ**](https://github.com/mcpt/WLMOJ) for more judging backend details.
```
Comment (Ninjaclasher) [inline README.md:90]: `mcpt/WLMOJ` -> `MCPT/wlmoj`

**Comment by 6167656e74323431** [comment]
This is a horrible idea. The only meaningful thing this adds is the potential for future merge conflicts.

**Review by fluix-dev — CHANGES_REQUESTED** [review]
All images should be changed for consistency too, if this will even happen.

**Comment by UserBlackBox** [comment]
image

**Comment by hexadecimalDinosaur** [comment]
This was merged from a fork that was deleted

`wants to merge 3 commits into mcpt:master from unknown repository`
